views/imagestream-listing.html

```diff
@@ -1,4 +1,4 @@
-<table class="listing-ct">
+<table ng-class="classNames ? '{{classNames}}' : 'listing-ct'">
```

Comment:
listing-ct can still be passed in as an optional class.  In the origin-web-console, the listing-ct classes actually break our desired layout, hence this approach; this gives the consumers of the repo direct control over which styles are used while maintaining backward compatibility.